File: Spring/SpringDataJPA_hello/src/main/java/com/sample/SpringDataJPA_hello/controller/HelloController.java
package com.sample.SpringDataJPA_hello.controller;

import com.sample.SpringDataJPA_hello.entity.Customer;
import com.sample.SpringDataJPA_hello.repository.CustomerRepository;
import org.springframework.beans.factory.annotation.Autowired;
import org.springframework.boot.autoconfigure.EnableAutoConfiguration;
import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.ResponseBody;
import org.springframework.web.bind.annotation.RestController;

@RestController
@EnableAutoConfiguration
public class HelloController {
    @Autowired
    CustomerRepository customerRepository;

    @RequestMapping("/")
    @ResponseBody
    public String index() {
        Customer created = customerRepository.save(
                new Customer(null, "Hidetoshi", "Dekisugi"));
        System.out.println(created + "is created.");
        customerRepository.findAll()
                .forEach(System.out::println);

        return "Hello world";
    }

}

File: Spring/SQSApp/src/main/kotlin/com/example/demo/HelloController.kt
package com.example.demo

import org.springframework.beans.factory.annotation.Autowired
import org.springframework.beans.factory.annotation.Qualifier
import org.springframework.cloud.aws.messaging.core.QueueMessagingTemplate
import org.springframework.http.HttpStatus
import org.springframework.web.bind.annotation.*

@RestController
@RequestMapping("api/hello")
class HelloController {
    @Autowired
    @Qualifier("sqsAppQueueMessaging")
    lateinit var sqsAppQueueMessaging: QueueMessagingTemplate

    @RequestMapping(method = [RequestMethod.GET])
    fun hello() = "hello world."

    @RequestMapping(method = [RequestMethod.POST])
    @ResponseStatus(HttpStatus.CREATED)
    fun postMessage(@RequestBody message: String): String {
        sqsAppQueueMessaging.convertAndSend(message)
        return "OK"
    }

}